### ISSUE TITLE
Error 404 on clicking search link.

### ISSUE THREAD

**Issue by ghost** [rskwan/bristool#2]
Whenever I click on the link it gives a 404 error.
![capture](https://user-images.githubusercontent.com/18366085/28152907-1aca00ae-67a3-11e7-9575-8dfa939a1d52.PNG)

**Comment by ghost** [comment]
Found the issue, you only need to change http://images.google.com/...... to http://google.com/......

**Comment by hashimaziz1** [comment]
> 
> 
> Found the issue, you only need to change http://images.google.com/...... to http://google.com/......

You're right, but the code needs to be changed to reflect this, otherwise it makes the whole site essentially useless.

@rskwan Any chance you could accept the PR I made? I think that should fix the issue, and the site is perfect otherwise.
